recipes/stashy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "stashy" %}
+{% set version = "0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8137932f8fec2400e555fb230d51a47deb389955d2a2dbb37c1b6e59cf5f5983
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be noarch: python?

Reply:
No, there's a preprocessor conditional for enum34 on the list of dependencies... :/

Reply:
Uggh ok